### ISSUE TITLE
Add flavor awareness in sentry.gradle source map script

### ISSUE THREAD

**Issue by jennmueng** [getsentry/sentry-react-native#1660]
Came up in slack where a customer wants to prevent auto release creation in specific flavors.

**Comment by marandaneto** [comment]
@jennmueng would that be something similar to Android Proguard? https://docs.sentry.io/platforms/android/proguard/#variant-filtering

**Comment by marandaneto** [comment]
cc @jennmueng

**Comment by jennmueng** [comment]
Yes, looks to be exactly like https://docs.sentry.io/platforms/android/gradle/#variant-filtering. @marandaneto

**Comment by marandaneto** [comment]
Consider doing https://github.com/getsentry/sentry-react-native/issues/1463 too

**Comment by marandaneto** [comment]
Its already possible to do some sort of config with `project.ext.sentryCli` on Gradle, so this one can fit in there.

**Comment by antonis** [comment]
Already implemented. packages/core/sentry.gradle:200-211 uses a flavorAware config to resolve sentry-${variant}.properties per variant.
